src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -546,6 +546,7 @@ public ModelingSubmission addModelingSubmissionWithResultAndAssessor(ModelingExe
         Result result = new Result();
         result.setSubmission(submission);
         result.setAssessor(getUserByLogin(assessorLogin));
+        result.setAssessmentType(AssessmentType.MANUAL);
```

Comment:
Why did you have to change this?

Reply:
The logic for the repository method that retrieves participations without a manual result was changed: https://github.com/ls1intum/Artemis/pull/956/files#diff-350726bc24113981e92decb23ba810a6R64.
The test case setup is incomplete because the Result in this case would have the type 'MANUAL'. Therefore the submission would still be considered 'unassessed'.